extensions/text.js

```diff
@@ -186,6 +213,33 @@
               },
             },
           },
+          {
+            opcode: "surround",
+            blockType: Scratch.BlockType.REPORTER,
+            text: Scratch.translate("surround [TEXT] with [QUOTE]"),
+            arguments: {
+              TEXT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "apple",
+              },
+              QUOTE: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "QUOTES",
+              },
+            },
+          },
```

Comment:
Is there any good reason for this to be added, it seems to just be a join block but more convoluted than needed.

Reply:
We could go without it, mostly was just made for fun and a little qol

Reply:
i do not like this block

Reply:
Me either, I think it should be removed.

Reply:
done